packages/nx-plugin/src/generators/component-playwright-spec/generator.ts

```diff
@@ -0,0 +1,20 @@
+import { getProjects, formatFiles, generateFiles, Tree } from '@nx/devkit';
+import * as path from 'path';
+import { PlaywrightComponentSpecGeneratorSchema } from './schema';
+
+export default async function (
+  tree: Tree,
+  options: PlaywrightComponentSpecGeneratorSchema
+) {
+  const { name } = options;
+  const projects = getProjects(tree);
+  const project = projects.get(name);
+  if (!project) {
+    return;
+  }
+
+  const { root: projectRoot } = project;
+
+  generateFiles(tree, path.join(__dirname, 'files'), projectRoot, options);
```

Comment:
we can simplify again

```suggestion
  const { name } = options;
  const projectConfig = readProjectConfiguration(tree, name);

  generateFiles(
    tree,
    path.join(__dirname, 'files'),
    projectConfig.root,
    options
  );
```